LocalPackages/SwiftUIExtensions/Sources/SwiftUIExtensions/ModalView/ModalView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ModalView.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import AppKit
+import SwiftUI
+
+public protocol ModalView: View {}
+
+public extension ModalView {
```

Comment:
Shared convenience protocol to extend our SwiftUI views so that they can be shown modally.  This `ModalView` adds support for showing modal alerts from our secondary targets, which may not have a sheet to rely on.